src/main/java/janggi/domain/Board.java

```diff
@@ -14,11 +16,24 @@ public final class Board {
 
     private boolean isTeam1Turn = true;
 
-    public Board(final Team team1, final Team team2) {
+    private Board(final Team team1, final Team team2, final boolean isTeam1Turn) {
```

Comment:
isTeam1Turn이 항상 true임을 의도한 것이라면, 생성자에서 제외함으로써 값이 변경될 여지를 없애는 것이 좋을 것 같네요!
```suggestion
    private Board(final Team team1, final Team team2) {
```

Reply:
맞네요! 항상 team1이 먼저하는 팀으로 설정되는게 맞습니다.

[refactor: 초기화가 필요없는 필드값을 생성자에서 제거](https://github.com/woowacourse/java-janggi/pull/86/commits/c02b383cefc3b09ba0040d6dc80e64605b5bb6a2)